src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,47 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+import java.time.Duration;
+
+public class LoginPage extends BasePage{
+        WebDriver driver;
+       WebDriverWait wait;
+
+        public LoginPage(WebDriver givenDriver) {
+                super(givenDriver);
+
+        }
+
+        public void enterEmail() {
+                WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/226164141-97fd26b4-6f12-44d8-9264-1358ec1fc3cf.png)

Reply:
Thank you,
are we supposed to remove the methods from BaseTest page after we move them to each new page ?